docusaurus/android_versioned_docs/version-6.x.x-before-refactoring/05-compose/05-message-components/05-attachments-picker.mdx

```diff
@@ -161,6 +161,7 @@ class AttachmentsPickerCustomTabFactory: AttachmentsPickerTabFactory {
 
     @Composable
     override fun PickerTabContent(
+        onAttachmentPickerAction: (AttachmentPickerAction) -> Unit,
```

Comment:
Same here. it's v6 before refactoring

Reply:
Fixed: [57354ef](https://github.com/GetStream/stream-chat-android/pull/5305/commits/57354efde216db50a3fc4a4ec85cce83f73fb0ec)